packages/app/utils/token.ts

```diff
@@ -1,8 +1,18 @@
 export const addressShortner = (address: string) =>
   address.slice(0, 4) + "..." + address.slice(-4);
 
-export const formatTokenValue = (value: number, custom?: number) => {
+export const formatTokenValue = (
+  value: number | string,
+  decimals: number = 4
+) => {
   if (value === 0) return value.toFixed(2);
```

Comment:
Oh, this is legacy code, I didn't touch it because it may be used in another place (as diogo did it).

What I'm doing is to check [inside the TokenRow](https://github.com/SwaprHQ/stackly-ui/pull/54/files#diff-44dd933316d2da92fe4ba3e95088dd98d164ade168363074c5c5a436656a469fR189) if I should display a `0` or a formatted value